src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/steps/VotingStep/VotingStep.tsx

```diff
@@ -1,10 +1,106 @@
 import React, { FC } from 'react';
+import { formatText } from '~utils/intl';
+import Button from '~v5/shared/Button';
+
+import CardWithStatusText from '~v5/shared/CardWithStatusText';
+import ProgressBar from '~v5/shared/ProgressBar';
+import { VotingStepItem } from './types';
+import Icon from '~shared/Icon';
+import TextWithValue from './partials/TextWithValue';
+import FormButtonRadioButtons from '~v5/common/Fields/RadioButtons/ButtonRadioButtons/FormButtonRadioButtons';
+import { ActionForm } from '~shared/Fields';
+import { ActionTypes } from '~redux';
 
 const displayName =
   'v5.common.ActionSidebar.partials.motions.MotionSimplePayment.steps.VotingStep';
 
 const VotingStep: FC = () => {
-  return <div>VotingStep</div>;
+  const items: VotingStepItem[] = [
+    {
+      key: '1',
+      text: formatText({ id: 'motion.votingStep.votingMethod' }),
+      children: <span className="text-sm">Reputation-weighted</span>,
+    },
+    {
+      key: '2',
+      text: formatText({ id: 'motion.votingStep.teamReputation' }),
+      children: (
+        <div className="flex items-center gap-1">
+          <Icon name="star" appearance={{ size: 'tiny' }} />
+          <span className="text-sm">17.61%</span>
+        </div>
+      ),
+    },
+    {
+      key: '3',
+      text: formatText({ id: 'motion.votingStep.rewardRange' }),
+      children: <span className="text-sm">0.0006 - 12 CLNY</span>,
```

Comment:
This text style is the same for each item - maybe you can add it inside the component?